config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
+  publishable_key: ENV['STRIPE_PUBLISHABLE_KEY'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.